.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
- - 5.5
```

Comment:
Well, phpunit 4.8 does not support PHP7 actually, which we support as well.
That's why I opted for going with phpunit 5.2 and drop php5.5 testing support, since 5.6 and 5.5 are more or less the same anyway.